src/DataTransferObject.php

```diff
@@ -17,14 +17,27 @@ abstract class DataTransferObject
 
     protected array $onlyKeys = [];
 
-    public function __construct(...$args)
+    public static function new(...$args): static
```

Comment:
This could potentially just be:

```php
public static function new(...$arguments): static
{
    return static::newWithoutValidation($arguments)->validate();
}
```